lua/ttt2/libraries/database.lua

```diff
@@ -465,7 +519,7 @@ clientReceiveFunctions[MESSAGE_REGISTER] = function(data)
 		keys = data.savingKeys,
 		data = data.additionalData,
 		storedData = {},
-		defaultData = {}
+		defaultData = {},
```

Comment:
Is this correct? A comma after the last entry?

Reply:
lua does allow this, yes.

We don't need to cripple ourselves only because of JSON being stupid :D

as for the reason why this is a good change: it removes the last-entry special case which causes unneeded friction when moving entries around or appending entries in the source. Both would otherwise lead to more changes than necessary (adding an removing `,`)

Reply:
imo, lua should always leave the last `,` because it decreases line noise in commits and make merges go cleaner

Reply:
fair point, you two convinced me with your arguments